Output.md

```diff
@@ -0,0 +1,156 @@
+vault_mount.common: Creating...
```

Comment:
This output doesn't offer much to readers, it's a default Terraform output for resource creation.